man/zypper-docker-lp.1.md

```diff
@@ -40,5 +40,8 @@ the running container is based on.
 **-g**, **--category**
   List only patches with this category.
 
+**--severity**
+  List only patches with this severity.
```

Comment:
I would add something like "note well: this requires zypper >= x.y.z inside of your docker image"